src/sysusers/sysusers.c

```diff
@@ -163,6 +168,48 @@ static void maybe_emit_login_defs_warning(Context *c) {
         c->login_defs_need_warning = false;
 }
 
+static void log_audit_accounts(Context *c, bool users) {
```

Comment:
maybe make the 2nd argument `ItemType type` instead? 

and then do 

```c
assert(IN_SET(type, ADD_USER, ADD_GROUP));
```

I think that's much nicer then using a boolean here, that isn#t conceptually really a boolean...